scripts/kotlin-lint-check

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
+
+status=$?
+
+if [ "$status" = 0 ] ; then
+  echo "Lint completed successfully."
+  exit 0
+else
+  echo "********************************"
+  echo "Ktlint issue found."
+  echo "Please fix the above issues. You could also use ktlint -F --android path/to/file.kt
```

Comment:
Please update the suggested ``ktlint`` command to just fix all issues in the codebase (e.g. by passing all of the module folders to it).

Reply:
done